ivadomed/models.py

```diff
@@ -716,7 +716,7 @@ def forward(self, x_mods, indexes_mod):
         return preds
 
 
-class UNet3D(nn.Module):
+class Modified3DUNet(nn.Module):
```

Comment:
I don't have a strong preference about it.. but I thought "Modified" was not really explicit, and there is heaps of modified version of unet. What about `Unet3D_Isensee` (first author of the cited paper) or `Unet3D_BraTS2017`?

Reply:
I used the name used in the original repo: https://github.com/pykao/Modified-3D-UNet-Pytorch/blob/master/model.py. Isensee did multiple models, same thing for BraTS2017. I feel like there are all to some extend not explicit. Modified3DUnet describes "best" the architecture of the model, but same as you I don't have a strong preference about it :}

Reply:
Yes, I thought it was the reason... Mmm yes not easy one. I guess as long as it is properly referenced in the docstring it should be alright.